server/auth/local/verificationToken.model.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+import {Schema} from 'mongoose';
+import uuid from 'node-uuid';
+var mongoose = require('bluebird').promisifyAll(require('mongoose'));
+
+// Verification token model
+var verificationTokenSchema = new Schema({
```

Comment:
Do we really need a separate database for this, rather than just adding it to the user database?